SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -1968,15 +1982,15 @@ FRPCErrorInfo USpatialReceiver::ApplyRPC(const FPendingRPCParams& Params)
 	TWeakObjectPtr<UObject> TargetObjectWeakPtr = PackageMap->GetObjectFromUnrealObjectRef(Params.ObjectRef);
 	if (!TargetObjectWeakPtr.IsValid())
 	{
-		return FRPCErrorInfo{ nullptr, nullptr, ERPCResult::UnresolvedTargetObject };
+		return FRPCErrorInfo{ nullptr, nullptr, ERPCResult::UnresolvedTargetObject, ERPCQueueCommand::StopProcessing };
```

Comment:
When does this conditional get hit / why would we want this response? (vs. deleting entire list)

Reply:
I am not an expert on this but... talking with Valentyn he suggested maybe there is a case with dynamic sub objects that would hit this condition. Perhaps the thought being we might later receive the sub object and process the RPC on it.